star-wars-browser/src/components/search_results/index.js

```diff
@@ -1,5 +1,20 @@
+import { connect } from 'react-redux';
 import SearchResultsModal from './SearchResults';
+import { getPageNumber } from '../../store/pagination/selectors';
+import { getPeopleData } from '../../store/peopleData/selectors';
+import { changePage, decrement, increment } from '../../store/pagination/actions';
+import { setPeopleData } from '../../store/peopleData/actions';
 
-export default SearchResultsModal;
+const mapStateToProps = (state) => ({
+  pageNumber: getPageNumber(state),
+  peopleData: getPeopleData(state),
+});
 
-//funkcja connect, mapStateToProps
\ No newline at end of file
+const mapDispatchToProps = (dispatch) => ({
+  increment: () => dispatch(increment()),
```

Comment:
increment i decrement chyba nie są wykorzystywane w komponencie? Jeśli nie są, to można je usunąć.